app/models/course.rb

```diff
@@ -40,6 +40,13 @@ class Course < ApplicationRecord
     further_education: "Further education",
   }, _suffix: :course
 
+  enum degree_classification_required: {
```

Comment:
I don't think we need the `required` here. Also, register calls this `degree_grade`.

If this is an enum shouldn't it be an integer column and then map `two_one: 0` etc?

Reply:
Just to clarify there, I wasn't suggesting this has to be named the same as register but it would be good to be consistent across 'all of the things :tm:' if we can.

Reply:
Yea no worries. Happy to change it to degree grade. I was just thinking it might get a bit funky down the line as degree_grade may be used in other places to collect a candidates degree grade rather than the required one to get onto a course.